data/dictionary.js

```diff
@@ -0,0 +1,8 @@
+export default {
+  form_name: 'Form',
+  case_note_title: 'Title',
+  date_of_event: 'Date of event',
+  timestamp: 'Date created',
+  worker_email: 'Created by',
+  case_note_description: 'Description',
```

Comment:
We already have these defined in the `data/forms`, in addition to be a duplication I can see that would go easily out of sync.